cmsl1t/plotting/efficiency.py

```diff
@@ -130,11 +142,25 @@ def __make_overlay(self, pileup, threshold, hists, fits, labels, header):
             legend.Draw()
 
             # Save canvas to file
-            name = self.filename_format.format(type="efficiency_",
-                                               pileup=pileup,
+            name = self.filename_format.format(pileup=pileup,
                                                threshold=threshold)
             self.save_canvas(canvas, name)
 
     def __summarize_fits(self):
         """ Implement this to show fit evolution plots """
         pass
+
+    def _is_consistent(self, new):
+        """
+        Check the two plotters are the consistent, so same binning and same axis names
+        """
+        return (self.pileup_bins.bins == new.pileup_bins.bins) and \
+               (self.thresholds.bins == new.thresholds.bins) and \
+               (self.online_name == new.online_name) and \
+               (self.offline_name == new.offline_name)
+
+    def _merge(self, right):
+        """
+        Combine another plotter with this one
```

Comment:
Again a small mismatch of comment and function/parameter.
The doc string reflects the usual naming of the second object:
```
def _merge(self, other):
```